autorest/codegen/models/dictionary_schema.py

```diff
@@ -69,3 +70,8 @@ def from_yaml(cls, yaml_data: Dict[str, str], **kwargs: Any) -> "DictionarySchem
             element_type=element_type,
             additional_properties=for_additional_properties
         )
+
+    def imports(self) -> FileImport:
+        file_import = FileImport()
```

Comment:
You probably wants to merge from your internal type, in case this schema would need a specific import too